src/pages/Patients.jsx

```diff
@@ -12,58 +13,54 @@ const Patients = () => {
       status: "IN PROGRESS IN PROGRESS",
     },
   ]);
-  const ActionButton = () => {
-    return <button>Re Assign</button>;
+  const ReAssignButton = () => {
```

Comment:
Try to refactor this to use a cell renderer instead of having multiple components in the same file. THis should preferably be a separate file. Reference https://www.ag-grid.com/react-data-grid/component-cell-renderer/

Reply:
Done